### ISSUE TITLE
Seeing stats overtime?

### ISSUE THREAD

**Issue by rachelombok** [vinceh121/socialblade-api#4]
I think this is more of a feature request than an issue, but is there any way to see subscribers/followers over time, like how the graphs/charts show them on social blade? Like for example getting the subscribers of a channel from each month for the last two years? Just curious if there is any feature like this in the API, or a documentation guide explaining this, thanks!

**Comment by vinceh121** [comment]
Yes, it is a feature and already implemented for Youtube and Twitter, and in all honesty I'm surprised it still works as the v2 SocialBlade API has been deprecated since June 1st 2018.

Please see the `YTStats#getDataDaily` API. Unfortunately the data doesn't go as far as two years, but what seems like around 4 months of the daily subscriber and total views count.

Hope this helps!
